client/my-sites/subscribers/tracks/use-record-remove-modal.ts

```diff
@@ -0,0 +1,14 @@
+import useRecordSubscriberTrackEvent from './use-record-subscriber-tracks-event';
+
+const useRecordRemoveModal = () => {
+	const recordSubscribersTracksEvent = useRecordSubscriberTrackEvent();
+
+	return ( paid: boolean, action: string ) => {
+		const eventName = `calypso_subscribers_remove_${
+			paid ? 'paid' : 'free'
+		}_subscriber_${ action }`;
```

Comment:
Could you add a comment above enumerating the list of tracks event this caters for? It's for dev searchability reasons.